### DISCUSSION TITLE
Receiving Errors while applying for Student Developer Pack

### DISCUSSION THREAD

**Discussion by ShivamSikotra11** [community/community#55293] [GitHub Education]
### Hi everyone,

While registering for the Student Developer PAck by GITHUB I have come across some isseues i.e.
I had already updated my name and  bio in my profile (Personal as well as Public both ) still it is showing that please update that.

**Here I am also uploading the Error which i came across :** 
![image](https://github.com/community/community/assets/107760405/869e1793-b2bf-4e7a-bc75-c210e6bcce6f)

**Here is my updated Profile as I discussed earlier:** 
![image](https://github.com/community/community/assets/107760405/6a8a07ad-128f-43aa-9892-2df550d017a1)

**Comment by ryanlua** [comment]
Hi @ShivamSikotra11,

It seems like the information your provided GitHub isn't accurate or recognized. Below is a list of common problems along with relevant links. 

- **Check if your school is recognized by GitHub** Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.
- **Use up to date documents/information.** Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.
- **Ensure your GitHub profile matches.** Make sure all information such as your name or location both match your profile and documents.
- **Don't use a VPN.** GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.
- **Translate your documents.** If your school is in another language than English than you should provide both the original untranslated document and English translated document.

You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections. 

As a reminder you do not need to have a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

I hope this helps you with your rejection.

**Comment by samus-aran** [comment]
Hi @ShivamSikotra11 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
